java/delete-features-feature-service/README.md

```diff
@@ -0,0 +1,32 @@
+# Delete features (Feature Service)
+
+Delete features from an online feature service.
+
+![Delete Features (Feature Service)](delete-features-feature-service.png)
+
+## Use case
+
+A `FeatureLayer` created using this `SeviceFeatutreTable` that is applied to the `ArcGISMap` will display existing Features and allow a user to delete individual features by tapping on them.
+
+## How it works
+
+1. Create a service feature table from a URL, `new ServiceFeatureTable("URL")`
```

Comment:
Syntax in plain english is always funky, but we try to say more like:

Create a new `ServiceFeatureTable` from a url.

Same goes for below